pkg/api/v1alpha1/cluster_types.go

```diff
@@ -762,6 +766,126 @@ type KindAccessor interface {
 	ExpectedKind() string
 }
 
+// PackageConfiguration for installing EKS Anywhere curated packages.
+type PackageConfiguration struct {
+	// Disable package controller on cluster
+	Disable bool `json:"disable,omitempty"`
+
+	// Controller package controller configuration
+	Controller PackageControllerConfiguration `json:"controller,omitempty"`
+
+	// Cronjob for ecr token refresher
+	CronJob PackageControllerCronJob `json:"cronjob,omitempty"`
+}
+
+// Equal for PackageConfiguration.
+func (n *PackageConfiguration) Equal(o *PackageConfiguration) bool {
+	if n == o {
+		return true
+	}
+	if n == nil || o == nil {
+		return false
+	}
+	return n.Disable == o.Disable && n.Controller.Equal(&o.Controller)
```

Comment:
we don't consider `CronJob` in the equality?

Reply:
Added, thanks.